MPSGameLift/Code/mpsgamelift_client_files.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(FILES
+    Source/MPSGameLiftClientSystemComponent.cpp
+    Source/MPSGameLiftClientSystemComponent.h
+)
```

Comment:
nitpick: missing newline at the end of this file.

Reply:
Fixed